godtools/Managers/DownloadResources/DownloadedResourceManager.swift

```diff
@@ -11,14 +11,14 @@ import UIKit
 import Alamofire
 import PromiseKit
 import Spine
-import MagicalRecord
+import RealmSwift
 
 class DownloadedResourceManager: GTDataManager {
     static let shared = DownloadedResourceManager()
     
     let path = "/resources"
     
-    var resources = [DownloadedResource]()
+    var resources = List<DownloadedResource>()
```

Comment:
I believe you can set `typealias DownloadResources = List<DownloadedResource>` outside of the class, so you can use that smaller `DownloadResources` everywhere instead of using `List<DownloadedResource>`